bin/nvcc_wrapper

```diff
@@ -12,8 +12,8 @@
 # or g++ as their back-end compiler.  The defaults can be overwritten
 # by using the usual arguments (e.g., -arch=sm_30 -ccbin icpc).
 
-default_arch="sm_35"
-#default_arch="sm_50"
+default_arch="sm_70"
+#default_arch="sm_80"
```

Comment:
Your PR is based on a version of from Aug 20, 2020
[5f931d4](https://github.com/kokkos/kokkos/commit/5f931d450026abbdb283ad8f30764601202db35a)

This change is already in the current `develop`.
https://github.com/kokkos/kokkos/blob/c5fe10e7f9e4925f0269f1eb155770935117f2d4/bin/nvcc_wrapper#L17-L18

Please fetch the latest version and re-apply your patch on top of it.
I can assist if needed.

Reply:
Sorry about that ... I did a fetch to update the fork and thought it updated. Let me go back and see what happened.